### PR TITLE
chore: add volunteer ShaikMoAbdullah

### PR DESCRIPTION
**Pull request by ShaikMoAbdullah**
chore: add ShaikMoAbdullah profile to AskDevs

### CODE REVIEW

File: data/users/ShaikMoAbdullah.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Shaik Mohammad Abdullah",
+  "bio": "JavaScript Developer | Web Dev | React Dev",
```
Comment (codewithvoid) [inline data/users/ShaikMoAbdullah.json:3]: pick 1 title. `Web Developer` preferably

File: data/users/ShaikMoAbdullah.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  "image": "https://drive.google.com/file/d/1r4aHtrJWeIKD4HuKArl4MHQFPjXbbwHR/view?usp=sharing",
```
Comment (codewithvoid) [inline data/users/ShaikMoAbdullah.json:4]: can you use github url instead
https://avatars.githubusercontent.com/u/52700318?v=4

File: data/users/ShaikMoAbdullah.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  "categories": [
+    {
```
Comment (codewithvoid) [inline data/users/ShaikMoAbdullah.json:6]: you can only use 1 slug per category.
```json
[
  {
    "slug": "programming",
    "topics": [
      "JavaScript"
    ]
  },
  {
    "slug": "web-development",
    "topics": [
      "HTML",
      "CSS",
      "ReactJS"
    ]
  }
]
```

File: data/users/ShaikMoAbdullah.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+      "slug": "programming | software-development | web-development",
+      "topics": ["JavaScript", "React", "HTML and CSS"]
+    }
+  ],
+  "contacts": {
+    "twitter": "ShaikMoAbdullah>"
```
Comment (codewithvoid) [inline data/users/ShaikMoAbdullah.json:12]: remove `>`
```json
"twitter": "ShaikMoAbdullah"
```

**Review by codewithvoid — CHANGES_REQUESTED** [review]
Thanks @ShaikMoAbdullah for volunteering. Can you please address the comments?

File: data/users/ShaikMoAbdullah.json (hunk 1 above)
Comment (ShaikMoAbdullah) [reply data/users/ShaikMoAbdullah.json]: Made the changes

File: data/users/ShaikMoAbdullah.json (hunk 2 above)
Comment (ShaikMoAbdullah) [reply data/users/ShaikMoAbdullah.json]: Made the changes

File: data/users/ShaikMoAbdullah.json (hunk 3 above)
Comment (ShaikMoAbdullah) [reply data/users/ShaikMoAbdullah.json:6]: Made the changes

File: data/users/ShaikMoAbdullah.json (hunk 4 above)
Comment (ShaikMoAbdullah) [reply data/users/ShaikMoAbdullah.json]: Made the changes

**Comment by ShaikMoAbdullah** [comment]
Thanks for the feedback

File: data/users/ShaikMoAbdullah.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  "bio": "Web Developer",
+  "image": "https://avatars.githubusercontent.com/u/52700318?v=4",
+  "categories": [
+    {
+      "slug": "programming",
+      "topics": ["JavaScript"]
+    },
+    {
+      "slug": "software-development",
+      "topics": ["JavaScript", "React", "HTML and CSS"]
```
Comment (codewithvoid) [inline data/users/ShaikMoAbdullah.json:12]: you can remove software-development category

File: data/users/ShaikMoAbdullah.json [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+    },
+    {
+      "slug": "web-development",
```
Comment (codewithvoid) [inline data/users/ShaikMoAbdullah.json:15]: you have already mentioned JavaScript in programming. So just mention others here. Example
```json
  {
    "slug": "web-development",
    "topics": [
      "HTML",
      "CSS",
      "ReactJS"
    ]
  }
```

**Review by codewithvoid — CHANGES_REQUESTED** [review]
Thanks @ShaikMoAbdullah. Can you resolve 2 more changes.

File: data/users/ShaikMoAbdullah.json (hunk 5 above)
Comment (ShaikMoAbdullah) [reply data/users/ShaikMoAbdullah.json]: Removed

File: data/users/ShaikMoAbdullah.json (hunk 6 above)
Comment (ShaikMoAbdullah) [reply data/users/ShaikMoAbdullah.json:11]: Updated

**Review by codewithvoid — APPROVED** [review]
Thanks for making the changes @ShaikMoAbdullah. Welcome!

**Pull request merged**